Makefile

```diff
@@ -348,3 +348,13 @@ operator-lint: gowork ## Runs operator-lint
 .PHONY: docs
 docs: ## Create documentation under docs/build/html
 	docs/build-docs.sh
+
+BRANCH=main
+.PHONY: force-bump
+force-bump: ## Force bump operator and lib-common dependencies
+        for dep in $$(cat go.mod | grep openstack-k8s-operators | grep -vE -- 'indirect|test-operator|^replace' | awk '{print $$1}'); do \
```

Comment:
from https://github.com/openstack-k8s-operators/test-operator/actions/runs/14390474791/job/40356244806?pr=294 `Makefile:355: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.`